### ISSUE TITLE
Undo-redo stack is getting disordered

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#6635]
### What type of issue is this?

Transient – Occurring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

We were working for a longer time with a project (can attach later a video from a full workflow if needed):
1. opened an existing project
2. added few nodes at the beginning of the workflow
3. worked a little with the graph
4. moved down the graph
5. disconnected two nodes
6. another node was moved
7. we tried to undo last actions - it took us 7 undo to get the connection back
8. before we got to reconnecting the node, the nodes we added at the beginning of the workflow were removed

### Expected behaviour

Right order in the undo stack

### Enso Version

develop 10.05

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

standalone

### Operating System

Windows

**Comment by Akirathan** [comment]
I tried to reproduce it on larger project `Enso_201` and I am unable to reproduce it. Video would be nice, if you have some.

**Comment by farmaazon** [comment]
I think this should be considered a part of #6435. @mwu-tow please close this one if I'm right.

**Comment by farmaazon** [comment]
Closing, as we use different tool for undo redo in GUI2. Any issue with it should be reported as a new task.
